README.md

```diff
@@ -212,6 +212,20 @@ and `tenant_id` parameters must be specified. The `tenant_id` is typically the *
 as defined by HP.
 
 
+### Amazon S3 ###
+
+A reference to an object in an Amazon S3 bucket.  The `s3` scheme can be used when storing
+files using the Amazon S3 service. 
```

Comment:
You should document that:

* the region parameter is required
* Downloading currently doesn't use chunked encoding, so it shouldn't be used for large files.